test/e2e/framework/pv/pv.go

```diff
@@ -53,6 +52,14 @@ const (
 
 	// VolumeSelectorKey is the key for volume selector.
 	VolumeSelectorKey = "e2e-pv-pool"
+
+	// IsDefaultStorageClassAnnotation represents a StorageClass annotation that
+	// marks a class as the default StorageClass
+	IsDefaultStorageClassAnnotation = "storageclass.kubernetes.io/is-default-class"
```

Comment:
Will we use this IsDefaultStorageClassAnnotation from outside of this package?
If not, we can define this as `isDefaultStorageClassAnnotation` as a local one to avoid confusion related to duplicated one with the original one.

Reply:
fixed, thanks